huaweicloud/services/drs/resource_huaweicloud_drs_job.go

```diff
@@ -815,6 +880,17 @@ func flattenPolicyConfig(detail jobs.JobDetail) []interface{} {
 	return rst
 }
 
+func flattenAlarmNotify(alarmNotify jobs.AlarmNotifyInfo) []interface{} {
```

Comment:
this will make the topic_urn empty, and there will always be changing plan

Reply:
`topic_urn` is not in return, it's stated in docs that must ignore its change.